src/main.rs

```diff
@@ -241,8 +240,9 @@ fn main() {
         Some(PathBuf::from(prefix_dir))
     };
 
-    let tmp_dir = TempDir::new("grcov").expect("Failed to create temporary directory");
+    let tmp_dir = tempfile::tempdir().unwrap();
```

Comment:
Keep `expect` instead of `unwrap` so we have an error message should this fail (`except` is basically the same as `unwrap`, but with an error message).